### PR TITLE
Fix versioning issue and missing names | CU-2hr79nx

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: Ashish Tiwari <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [ ] Yes, I signed my commits.

### CODE REVIEW

File: build/config.go [hunk 1]
```diff
@@ -9,16 +11,23 @@ import (
 	"github.com/layer5io/meshery-consul/internal/config"
 
 	"github.com/layer5io/meshkit/utils"
+	"github.com/layer5io/meshkit/utils/kubernetes"
 	"github.com/layer5io/meshkit/utils/manifests"
 	smp "github.com/layer5io/service-mesh-performance/spec"
 )
 
+type Versions struct {
```
Comment (leecalcote) [inline build/config.go:19]: Other adapters seem to be missing this struct. As we drive toward consistency across adapters, is there any reason that this struct is not standard?

Comment (Revolyssup) [reply build/config.go:19]: This is more like a utility struct for a very specific use case. This is not needed anywhere else.

Comment (leecalcote) [reply build/config.go:19]: I politely and firmly reject this notion. Give me a ring.

**Review by leecalcote — APPROVED** [review]
Merge temp fix pending move to Meshery Adapter Library.

**Pull request merged**